File: semestr_2/lista3_lab/zad2/Pieciokat.cpp
#include "Pieciokat.hpp"

double Pieciokat::obliczPole()
{
    return bok * bok * std::sqrt(25 + 10 * std::sqrt(5)) / 4;
}
double Pieciokat::obliczObwod()
{
    return 5 * bok;
}

File: semester2/programming/list3/zad1/Pieciokat.java
import java.lang.Math;

// kat wewnetrzny = 108


public class Pieciokat implements Figura {
    public double obliczObwod()
    {
        return 5 * bok;
    }
    public double obliczPole()
    {
        return bok * bok * Math.sqrt(25 + 10 * Math.sqrt(5)) / 4;
    }

    public Pieciokat(double _bok)
    {
        bok = _bok;
    }

    private double bok;
}
